### DISCUSSION TITLE
Request for Commercial License to Avoid GPL-3.0 Issues

### DISCUSSION THREAD

**Discussion by T-nesh30** [ultralytics/ultralytics#15693] [Q&A]
Hello Ultralytics Team,

I am using YOLOv8 in a commercial product and are concerned about the GPL-3.0 license requiring us to open-source our code. Is there a commercial license available that would allow us to keep our code proprietary? We’d appreciate details on pricing and terms.

Thanks!

**Comment by glenn-jocher** [comment]
@T-nesh30 hello,

Yes, we offer an Ultralytics Enterprise License that allows you to use YOLOv8 in commercial products without the open-source requirements of the AGPL-3.0 license. Please visit https://ultralytics.com/license for more details on pricing and terms.

Thanks!
